src/uct/ib/base/ib_md.c

```diff
@@ -11,13 +11,49 @@
 
 #include <ucs/arch/atomic.h>
 #include <pthread.h>
+#include <numaif.h>
```

Comment:
we check the presence of the library with AC_CHECK_LIB. Do we need to check headers as well?